### DISCUSSION TITLE
Vercel Forms? (Like Netlify?)

### DISCUSSION THREAD

**Discussion by BraydenTW** [vercel/vercel#5945] [Ideas]
Hi,

I thought it would be pretty helpful for a ready-made form submission and collection tool to be included in Vercel. 

An example of this already through the works is Netlify Forms. It is super easy to use in a website and I think it would be a cool feature to add.

**Comment by mylesthedev** [comment]
This would be a great solution for those simple sites that usually have a short life span (marketing events).
